roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -5,94 +5,6 @@
 - name: Helm | Set up helm launcher
   include_tasks: "install_{{ helm_deployment_type }}.yml"
 
-- name: Helm | Lay Down Helm Manifests (RBAC)
-  template:
-    src: "{{ item.file }}.j2"
-    dest: "{{ kube_config_dir }}/{{ item.file }}"
-  with_items:
-    - {name: tiller, file: tiller-namespace.yml, type: namespace}
-    - {name: tiller, file: tiller-sa.yml, type: sa}
-    - {name: tiller, file: tiller-clusterrolebinding.yml, type: clusterrolebinding}
-  register: manifests
-  when:
-    - dns_mode != 'none'
-    - inventory_hostname == groups['kube-master'][0]
-
-- name: Helm | Apply Helm Manifests (RBAC)
-  kube:
-    name: "{{ item.item.name }}"
-    namespace: "{{ tiller_namespace }}"
-    kubectl: "{{ bin_dir }}/kubectl"
-    resource: "{{ item.item.type }}"
-    filename: "{{ kube_config_dir }}/{{ item.item.file }}"
-    state: "latest"
-  with_items: "{{ manifests.results }}"
-  when:
-    - dns_mode != 'none'
-    - inventory_hostname == groups['kube-master'][0]
-
-# Generate necessary certs for securing Helm and Tiller connection with TLS
-- name: Helm | Set up TLS
-  include_tasks: "gen_helm_tiller_certs.yml"
-  when: tiller_enable_tls
-
-- name: Helm | Install client on all masters
```

Comment:
Maybe this task should not be removed but modified to simply add stable repo when helm_stable_repo_url is defined?

Reply:
I can't get a point of this. If it doesn't need tiller then what for it repo in chart? I think to just delete `helm_stable_repo_url` in next commit

Reply:
In pre-v3, `helm init` not only installed tiller but also initiated the helm client on members of the kube-master group (optionally including adding stable repo). That made helm directly usable to install charts from stable repo on masters for users who chose to enable `helm_stable_repo_url` in their inventory. 
Your changes make the current behaviour change.